src/winit_input_helper.rs

```diff
@@ -319,28 +330,45 @@ impl WinitInputHelper {
         }
     }
 
-    /// Returns `None` when the mouse is outside of the window.
-    /// Otherwise returns the mouse coordinates in pixels.
-    pub fn mouse(&self) -> Option<(f32, f32)> {
+    /// Returns `None` when the cursor is outside of the window.
+    /// Otherwise returns the cursor coordinates in pixels.
+    pub fn cursor(&self) -> Option<(f32, f32)> {
         match &self.current {
-            Some(current) => current.mouse_point,
+            Some(current) => current.cursor_point,
             None => None,
         }
     }
 
-    /// Returns the change in mouse coordinates that occured during the last step.
+    /// Returns the change in cursor coordinates that occured during the last step.
     /// Returns `(0.0, 0.0)` if the window loses focus.
-    pub fn mouse_diff(&self) -> (f32, f32) {
+    pub fn cursor_diff(&self) -> (f32, f32) {
         if let Some(current_input) = &self.current {
-            if let Some(cur) = current_input.mouse_point {
-                if let Some(prev) = current_input.mouse_point_prev {
+            if let Some(cur) = current_input.cursor_point {
+                if let Some(prev) = current_input.cursor_point_prev {
                     return (cur.0 - prev.0, cur.1 - prev.1);
                 }
             }
         }
         (0.0, 0.0)
     }
 
+    /// Returns the change in mouse coordinates that occured during the last step.
+    ///
+    /// This function tracks device motion and tracks the
+    /// `DeviceEvent::CursorMotion` event. Unlike `cursor_diff`, this works
```

Comment:
> this works even if the window loses focus

This is not how the implementation in this PR behaves.
If the window loses focus `self.current` is set to None so `mouse_diff` will return `(0.0, 0.0)`

However I much prefer the behavior in the PR over the documented behavior as it matches the rest of the API exposed by winit_input_helper and I cant imagine ever wanting to actually receive such inputs while the window is out of focus.
So I will just update the docs to match reality.